src/components/input-date-picker/input-date-picker.e2e.ts

```diff
@@ -257,6 +257,25 @@ describe("calcite-input-date-picker", () => {
     expect(await calendar.isVisible()).toBe(true);
   });
 
+  it("allows clicking a date in the calendar popup", async () => {
+    const page = await newE2EPage();
+    await page.setContent(`<calcite-input-date-picker value="2023-01-31"></calcite-input-date-picker>`);
+    const inputDatePicker = await page.find("calcite-input-date-picker");
+
+    await inputDatePicker.callMethod("setFocus");
+    await page.waitForChanges();
+
+    const newValue = await page.evaluate(() => {
+      const inputDatePicker = document.querySelector("calcite-input-date-picker");
+      const datePicker = inputDatePicker.shadowRoot.querySelector("calcite-date-picker");
+      const datePickerMonth = datePicker.shadowRoot.querySelector("calcite-date-picker-month");
+      const datePickerDay = datePickerMonth.shadowRoot.querySelector("calcite-date-picker-day");
+      datePickerDay.click();
+      return inputDatePicker.value;
```

Comment:
Nitpick: could you add some visual separation between the click and value being returned? Alternatively, you could get the value outside of `evaluate` via the `E2EElement` APIs when asserting.

Reply:
using the E2EElement method worked great!